winum.el

```diff
@@ -447,7 +447,7 @@ windows, however a higher number can be reserved by the user-defined
        (vconcat window-vector
                 (make-vector (1+ (- number window-vector-length)) nil))))))
 
-(defun winum--window-list ()
+(defun winum--get-window-list ()
```

Comment:
I don't think this should be renamed `*--get-*`, because it is not a getter. Maybe if the name was to be changed it would be more like for `winum--list-windows` to be more consistent with `winum--list-windows-in-frame` and take the active voice.